components/Button/button.html

```diff
@@ -0,0 +1,356 @@
+              <a class="btn-primary-outline-non deco-none" href="./../../components/Card/card.html">Card</a>
+            </li>
+            <li class="pad-xs list-type-none">
+              <span class="iconify" data-icon="bi:file-earmark-image"></span>
+              <a class="btn-primary-outline-non deco-none" href="./../../components/Image/image.html">Image</a>
+            </li>
+            <li class="pad-xs list-type-none">
+              <span class="iconify" data-icon="whh:html"></span>
+              <a class="btn-primary-outline-non deco-none" href="./../../components/Input/input.html">Input</a>
+            </li>
+            <li class="pad-xs list-type-none">
+              <span class="iconify" data-icon="vaadin:modal-list"></span>
+              <a class="btn-primary-outline-non deco-none" href="./../../components/Modal/modal.html">Modal</a>
+            </li>
+            <li class="pad-xs list-type-none">
+              <span class="iconify" data-icon="fluent:navigation-16-regular"></span>
+              <a class="btn-primary-outline-non deco-none" href="#">Navigation</a>
+            </li>
+            <li class="pad-xs list-type-none">
+              <span class="iconify" data-icon="ic:twotone-star-rate"></span>
+              <a class="btn-primary-outline-non deco-none" href="#">Rating</a>
+            </li>
+            <li class="pad-xs list-type-none">
+              <span class="iconify" data-icon="bi:grid-1x2"></span>
+              <a class="btn-primary-outline-non deco-none" href="./../../components/Simplified Grid/simplified-grid.html">Simplified Grid</a>
+            </li>
+            <li class="pad-xs list-type-none">
+              <span class="iconify" data-icon="octicon:typography-24"></span>
+              <a class="btn-primary-outline-non deco-none" href="./../../components/Typography/typography.html">Typography</a>
+            </li>
+          </ul>
+        </div>
+      </aside>
+      <div class="main-content content-wrap">
+        <div class="text-5 highlightMainText">Button</div>
+        <hr class="mar-y-3" />
+        <p class="text-3 mar-y-2">
+          The Button component provides different clickable buttons, which are used to submit forms or anywhere in a document for accessible, standard button functionality.
+        </p>
+        <p class="mar-y-2 text-3">
+          The Button component comes in different states such as -<span class="imp-highlight-text sm-bdr-radius">primary</span>,<span class="imp-highlight-text sm-bdr-radius">secondory</span>,<span class="imp-highlight-text sm-bdr-radius">success</span>,<span class="imp-highlight-text sm-bdr-radius">warning</span>,<span class="imp-highlight-text sm-bdr-radius">danger</span>,<span class="imp-highlight-text sm-bdr-radius">dark</span>,<span class="imp-highlight-text sm-bdr-radius">light</span>,<span class="imp-highlight-text sm-bdr-radius">info</span> and <span class="imp-highlight-text sm-bdr-radius">link</span>.
```

Comment:
Again some formatting issue, maybe prettier extension not working?

Reply:
Yes I did